modules/es-extensions/store/extensions/app/greg-store-defaults/asset.js

```diff
@@ -85,14 +85,18 @@ asset.renderer = function(ctx) {
                     page.notifications = gregAPI.notifications.list(am);
                 }
             },
-        	checkDependents:function(page) {
-        		if(page.assets){
-        			var dependencies  = page.assets.dependencies || [];
-        			var dependents = page.assets.dependents || [];
-        			var isDependentsPresent =  ( dependencies.length > 0 ) || (dependents.length > 0 );
-        			page.assets.isDependentsPresent = isDependentsPresent;
-        		}
-        	},
+            checkDependents: function (page) {
+                if (page.assets) {
+                    var dependencies = page.assets.dependencies || [];
+                    var dependents = page.assets.dependents || [];
+                    var dependencyCheck = {
+                        isDependencies: dependencies.length > 0,
+                        isDependents: dependents.length > 0
+                    };
+                    dependencyCheck.isAnyDependence = ( dependencyCheck.isDependencies ) || ( dependencyCheck.isDependents );
```

Comment:
Thanks, I have fixed it in [this commit](https://github.com/wso2/product-greg/pull/854/commits/ccc776caee20c9938a269014cd7da7544e39dc3a)